docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -633,5 +633,6 @@ Note that you might have conflicts between previously installed PyPI dependencie
 For the models in ONNX* format, there are two available paths of IR conversion. 
 The old one is handled by the old Python* implementation, while the new one uses new C++ frontends. 
 Starting from the 2022.1 version, the default IR conversion path for ONNX models is processed using the new ONNX frontend. 
-Certain features, such as `--extensions` and `--transformations_config`, are not yet supported on the new frontends. 
+Certain features, such as `--extensions and` `--transformations_config`, are not yet fully supported on the new frontends. 
+For `--extensions`, only paths to shared libraries (.dll and .so) are supported. For `--transformations_config`, MO supports JSON configurations with defined library fields. 
```

Comment:
Propose to clarify that this is about new frontends
```
For `--extensions`, the new frontends support only paths to shared libraries (.dll and .so).
For `--transformations_config`, they supports JSON configurations with defined library fields.
```

Reply:
done